api/external/data_feed/data_feed.proto

```diff
@@ -36,6 +77,17 @@ service DatafeedService {
 			action: "destination:destination:delete"
 		};
 	};
+/*
+  Update a datafeed destination
+
+  Updates a datafeed destination's `name`, `url` or `secret`.
```

Comment:
@satellite15 would this updated interpretation be correct then?

Reply:
yes